src/scss/_forms.scss

```diff
@@ -147,6 +147,20 @@ select.form-control-lg {
 }
 // scss-lint:enable QualifyingElement
 
+.form-control-success,
```

Comment:
Adding .form-control-success/warning/danger to this selector isn't necessary, since these styles are already set on these classes in Bootstrap.